docusign-connector-product/README.md

```diff
@@ -9,67 +16,102 @@ Axon Ivy’s [DocuSign](https://www.docusign.com/products/electronic-signature)
+   if you are being asked to authorize yourself with a docuSign account.  
    ![docusign-auth](images/docuSign_auth.png)
 
+### Variables
+
+In order to use this product you must configure multiple variables.
+
+Add the following block to your `config/variables.yaml` file of our 
+main Business Project that will make use of this product. Afterwards
+set the values that you collected before.
+(Note, that in the Designer these variables can be defined
+in any project so there is no need to unpack the demo project.)
+
+```
+Variables:
+  docusign-connector:
+    # Integration key from your applications settings in the DocuSign eSignature "Apps and Keys" page.
+    integrationKey: ''
+    
+    # Secret key from your applications settings in the DocuSign eSignature "Apps and Keys" page.
+    # [password]
+    secretKey: ''
+    
+    # If set, use a specific account id, otherwise use the default account of the user. (Probably only makes sense for JWT Token grant.)
+    accountId: ''
+
+    # Scope of grant.
+    scope: signature impersonation
+    
+    # Docusign base url for authentication.
+    baseUri: https://account-d.docusign.com/oauth
+
+    jwt:
+      # If 'true' JWT token grant else user grant (default).
+      use: false
+
+      # User ID from your eSignature "Apps and Keys" page.
+      userId: ''
+    
+      # Name of the key file from your applications settings in the DocuSign eSignature "Apps and Keys" page relative to the "configuration" directory.
+      keyFile: 'docusign.pem'
+```
 
 ### Optional: Allow System Authentication (JWT)
 
-The Demo process contains a final service part, where the Axon Ivy platform acts in the name of a user.
+The Demo process contains a final service part, where the Axon Ivy platform acts in the name of a user.  
 ![docusign-props](images/systemDrivenProcess.png)
 
 This interaction requires a JSON Web Token (JWT) authentication setup:
 
 1. Edit the docuSign `application` as in step 3. of the general setup.
-1. In the section `Authentication` click on `Generate RSA` in order to create a secure key-pair.
+1. In the section `Authentication` click on `Generate RSA` in order to create a secure key-pair.  
  ![docusign-gen-rsa](images/authenticationGenerateRSA.png)
 
 1. Store the generated private key:
 	1. Copy the generated 'Private Key' to your clipboard
 	1. Save the changed application settings
 	1. Create a new empty text file called `docusign.pem` in your Designer 'configuration' directory
 	1. Paste the contents of your clipboard into the `docusign.pem` file
-	1. You can use another storage location for the pem file. Adjust the Rest Client Property `AUTH.systemKeyFile` to refer it. It should be a relative path to the 'configuration' directory or an absolute path on your system.
+	1. You can use another storage location for the pem file. Adjust the variable `jwt.keyFile` to refer it. It should be a relative path to the 'configuration' directory or an absolute path on your system.  
```

Comment:
not sure about the variable name; I'd add the prefix to it which is in charge:
`docusign-connector.jwt.keyFile`

Reply:
ok